x/treasury/keeper/grpc_query.go

```diff
@@ -28,3 +29,25 @@ func (k Keeper) RelayerFee(ctx context.Context, req *types.QueryRelayerFeeReques
 
 	return f, nil
 }
+
+func (k Keeper) RelayerFees(ctx context.Context, req *types.QueryRelayerFeesRequest) (*types.QueryRelayerFeesResponse, error) {
+	i, err := k.GetRelayerFeesByChainReferenceID(ctx, req.ChainReferenceId)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get relayer fees: %w", err)
+	}
+
+	response := make([]types.RelayerFeeSetting, 0, len(i))
+	for _, key := range maps.Keys(i) {
```

Comment:
It might not be relevant here, but I think this is just sidestepping the linter.
From https://pkg.go.dev/golang.org/x/exp/maps#Keys
```
The keys will be in an indeterminate order.
```
We should sort them if we really must.

Reply:
It's not even sidestepping it unfortunately. My first attempt was to sort the keys, but for that we also have to iterate the map, even though it won't be consensus breaking. But CodeQL has no way of disabling warnings with annotations (yet). There's some CI extension that implements this, but I don't have the time to play around with it.